src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,105 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final int MINIMAL_AMOUNT_OF_SYMBOLS = 6;
+    private static final int MINIMAL_AMOUNT_OF_AGE = 18;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(null);
+        });
+    }
+
+    @Test
+    void register_DuplicateUsers_NotOk() {
+        User user1 = new User("Mykola", "1234567", 85);
+        Storage.people.add(user1);
+        User user2 = new User("Mykola", "1234567", 85);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(user2);
+        });
+    }
+
+    @Test
+    void check_IfMethodCanAddNewUser_Ok() {
```

Comment:
use the same method name format 
```suggestion
    void check_ifMethodCanAddNewUser_ok() {
```

Reply:
Not resolved